packages/components/pages/f-contact-preferences/src/services/mapping/contactPreferences.mapper.js

```diff
@@ -0,0 +1,56 @@
+import {
+    NON_EDITABLE_PREFERENCE_KEYS,
+    NON_VISIBLE_PREFERENCE_KEYS
+} from '../../constants';
+
+export const mapToPreferencesViewModel = ({ Preferences, PreferenceVersionViewed }) => {
+    const model = {
+        preferences: Preferences.map(({
+            DisplayName, Sort, Key, Push, Email, Sms
```

Comment:
Also is there a requirement for the keys to be capitalised?

Reply:
These are the actual fields inside the api response, so not internal fields

Reply:
Would you be able to line break each key so it reads like:

```
DisplayName,
Sort,
Key....